dev/devicelab/bin/tasks/flutter_attach_test_android.dart

```diff
@@ -57,10 +57,14 @@ Future<void> testReload(Process process, { Future<void> Function() onListening }
   }
 
   await eventOrExit(listening.future);
-  await eventOrExit(ready.future);
+  await eventOrExit(ready.future).timeout(const Duration(seconds: 15), onTimeout: () {
```

Comment:
Why did this take more than 5 seconds in the devicelab so consistently?  That seems like a long time...
How long does it take on your machine?

Reply:
Partly discussed offline. I thought it was just logcat pipe from the device but that's only taking 500ms. 

Takes me ~8s to do a whole attach process. There's definitely some hot spots in the process. Filed https://github.com/flutter/flutter/issues/53313 to dig for low hanging fruits.